content/k8s/manual-app-deployment/nginx-installation/_index.md

```diff
@@ -5,7 +5,8 @@ tags:
 - kubernetes
 - nginx
 prerequisites:
-  hard: []
+  hard: 
+  - k8s/manual-app-deployment/certbot
```

Comment:
They actually don't need to have Certbot before installing Nginx, they can do this in a different order.

Maybe flag as as soft prereq? Or remove the prereqs?